### DISCUSSION TITLE
Dependency error to ControlzEx 6.0.0

### DISCUSSION THREAD

**Discussion by Bjorne78** [MahApps/MahApps.Metro#4502] [General]
I would like to update ControlzEx to version 6.0.0 and I get the following errormessage: 

Severity	Code	Description	Project	File	Line	Suppression State
Error		Unable to resolve dependencies. 'ControlzEx 6.0.0' is not compatible with 'MahApps.Metro 2.4.10 constraint: ControlzEx (>= 4.4.0 && < 6.0.0)'.				

Is it possible to change this constraint?
Thank you for your response.

**Comment by timunie** [comment]
It's not a bug but a question. Use discussion for it. 

2.x will not be compatible but 3.0 beta is. You can consume it from nightly builds feed.
